### DISCUSSION TITLE
Using multiple "--cache-from" arguments

### DISCUSSION THREAD

**Discussion by f1sherman** [docker/buildx#1801] [General]
Hello! Is buildx able to handle multiple "--cache-from" arguments? We have a situation where we'd like to fall-back to a cache repository with a different name if the preferred cache doesn't exist. We're using multiple "--cache-from" arguments currently but it's not clear to me whether that's actually working and/or whether the ordering of the arguments matters. Thank you!

**Comment by tonistiigi** [comment]
Yes, you can use multiple. BuildKit will match the longest cache chain. Order does not matter (IIRC). On multiple matches, creation date is looked at (newer cache is better), and the local cache has higher priority.

**Reply by f1sherman** [reply]
Thank you!
